xcode/Subconscious/Shared/Components/FirstRun/FirstRunDoneView.swift

```diff
@@ -10,20 +10,66 @@ import SwiftUI
 struct FirstRunDoneView: View {
     @ObservedObject var app: Store<AppModel>
     @Environment(\.colorScheme) var colorScheme
+    
+    private var status: ResourceStatus {
+        app.state.gatewayProvisioningStatus
+    }
+    
+    private var dottedLine: some View {
+        Line()
+            .stroke(style: StrokeStyle(
+                lineWidth: 3,
+                dash: [6, 3]
+            ))
+            .frame(height: 1)
+            .foregroundColor(.secondary.opacity(0.5))
+            .frame(width: 24)
+    }
+    
+    private var did: Did {
+        Did(app.state.sphereIdentity ?? "")
+            ?? Config.default.subconsciousGeistDid
+    }
 
     var body: some View {
         NavigationStack {
-            VStack {
+            VStack(spacing: AppTheme.padding * 4) {
                 Spacer()
-                VStack(spacing: AppTheme.unit3) {
-                    StackedGlowingImage(
-                        image: Image("ns_logo"),
-                        width: AppTheme.onboarding.heroIconSize,
-                        height: AppTheme.onboarding.heroIconSize
+                VStack(spacing: AppTheme.padding * 2) {
+                    Text(
+                        status == .succeeded
+                        ? "Connected!"
+                        : "Connecting to Noosphere..."
```

Comment:
"Connecting to Noosphere" seems not quite right to me in that it suggest a slow internet connection.

WDYT about "Creating your sphere"

Reply:
I like this. I also mapped out copy for the other cases (starting offline or encountering an error).

Reply:
Cases:

<img width="366" alt="Screenshot 2023-05-11 at 11 26 47 am" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/6a63d91c-016a-47e6-847f-87dd4c7d4f28">
<img width="320" alt="Screenshot 2023-05-11 at 11 26 29 am" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/cbebea8b-05fb-451f-8a97-3f1ed66b7a01">
<img width="364" alt="Screenshot 2023-05-11 at 11 26 18 am" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/985c38e8-8587-451c-bd53-1f72243f5e08">
<img width="377" alt="Screenshot 2023-05-11 at 11 25 42 am" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/2fd6bb89-a737-4a75-ae53-5d640dbd7f35">